packages/api/orm/entity/Portfolio.ts

```diff
@@ -0,0 +1,42 @@
+import { Application } from "./Application";
+import { CloudServiceProvider } from "../../models/CloudServiceProvider";
+import { Column, Entity, OneToMany } from "typeorm";
+import { ProvisionableEntity } from "./ProvisionableEntity";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Portfolio extends ProvisionableEntity {
+  @Column({ type: String, nullable: false, length: 100 })
+  name: string;
+
+  @Column({ type: String, nullable: true, length: 300 })
+  description: string;
+
+  @Column({ type: String, nullable: false })
+  owner: string;
+
+  @Column({ type: "enum", enum: CloudServiceProvider, nullable: false })
+  csp: CloudServiceProvider;
+
+  @Column({ type: String, nullable: false, array: true })
+  dodComponents: Array<string>;
+
+  @Column({ type: String, nullable: false, array: true })
+  portfolioManagers: Array<string>;
+
+  @OneToMany(() => TaskOrder, (taskOrder) => taskOrder.portfolio)
+  taskOrders: TaskOrder[];
+
+  @OneToMany(() => Application, (application) => application.portfolio)
+  applications: Application[];
+
+  // operators - simple-array or PortfolioOperator[] ?
+  @Column({ type: "simple-array", default: "" })
+  administrators: string[];
+
+  @Column({ type: "simple-array", default: "" })
+  contributors: string[];
```

Comment:
Technically we don't support access levels other than administrator for Portfolios at the API level, but if we want to go ahead and re-use these three fields via inheritance that's probably fine since the API wouldn't allow using the other two access levels anyway.

Reply:
Good point.  Will make a note in the inherited class.